stage_descriptions/sorted-sets-08-gd7.md

```diff
@@ -0,0 +1,70 @@
+In this stage, you'll add support for retrieving the score of a zset member.
+
+### The `ZSCORE` Command
+
+The `ZSCORE` command is used to query the score of a member of a sorted set. If the zset and the member both exist, the score of the member is returned as a RESP bulk string.
```

Comment:
Let's split this explanation into two: 

---

The `ZSCORE` command is used .... The score is returned as a RESP bulk string. Example usage: 

```bash
> ZADD zset_key 24.34 "one"
(integer) 1
> ZADD zset_key 90.34 "two"
(integer) 1
> ZSCORE zset_key "one"
"24.34"
```

If the member or sorted set specified in arguments don't exist, a RESP null bulk string (`$-1\r\n`) is returned: 

```
...
```